metricbeat/docs/modules/istio.asciidoc

```diff
@@ -11,21 +11,20 @@ This file is generated! See scripts/mage/docs_collector.go
 beta[]
 
 This is the Istio module.
-This module is compatible with versions before `1.5` of Istio where microservices architecture is used. If using
-versions prior to `1.5` then `mesh`, `mixer`, `pilot`, `galley`, `citadel` metricsets should be used where the Istio
-module collects metrics from the
-Istio https://istio.io/v1.4/docs/tasks/observability/metrics/querying-metrics/#about-the-prometheus-add-on[prometheus exporters endpoints].
 
-For versions after `1.5`, `istiod` and `proxy` metricsets can be used.
+When using Istio `1.4-`, then the `mesh`, `mixer`, `pilot`, `galley`, `citadel` metricsets should be used. The Istio
+module collects metrics from these
+Istio https://istio.io/v1.4/docs/tasks/observability/metrics/querying-metrics/#about-the-prometheus-add-on[prometheus exporters endpoints].
 
-`istiod` collects metrics directly from Istio Daemon while `proxy` collects from each of the proxy sidecars.
+When using Istio `1.5+`, the `istiod` and `proxy` metricsets should be used. The `istiod` collects metrics directly from Istio Daemon Prometheus Exporter while `proxy` collects from the Prometheus Exporter of each of the proxy sidecars.
```

Comment:
Maybe we can add that metrics can be found at https://istio.io/latest/docs/reference/config/metrics/.

Reply:
Please review sentence added below